format/private/formatter_binary.bzl

```diff
@@ -42,7 +42,7 @@ BUILTIN_TOOL_LABELS = {
 CHECK_FLAGS = {
     "buildifier": "-mode=check",
     "swiftformat": "--lint",
-    "prettier": "--check",
+    "prettier": "--check --log-level error",
```

Comment:
why error rather than warn?

Reply:
Ah, no particular reason, I just happen to have that level patched. I think warn is a better one. Will fix.